tests/data/pullback_diffeo_metric_data.py

```diff
@@ -0,0 +1,44 @@
+import geomstats.backend as gs
+from tests.data_generation import TestData
+
+RTOL = 1e-4
+ATOL = 1e-5
+
+
+class PullbackDiffeoCircleMetricTestData(TestData):
+    def diffeomorphism_is_reciprocal_test_data(self):
+        smoke_data = [
+            dict(
+                metric_args=[],
+                point=gs.array([[1, 0], [0.7648421873, -0.6442176872]]),
+                rtol=RTOL,
+                atol=ATOL,
+            ),
+        ]
+        return self.generate_tests(smoke_data)
+
+    def tangent_diffeomorphism_is_reciprocal_test_data(self):
+        smoke_data = [
+            dict(
+                metric_args=[],
+                point=gs.array(
+                    [
```

Comment:
For more robust testing it is better to use a number of points that is different than the number of dimensions. e.g. 3 points in these case instead of 2. It just ensures reshapes, broadcasting and similar functions are really working properly.

Reply:
Okay !